server/broadcast.go

```diff
@@ -303,6 +323,13 @@ func (sp *SessionPool) selectSessions(ctx context.Context, sessionsNum int) []*B
 				sess.SegsInFlight = nil
 				sp.lastSess = removeSessionFromList(sp.lastSess, sess)
 				clog.V(common.DEBUG).Infof(ctx, "Removing orch=%v from manifestID=%s session list", sess.Transcoder(), sp.mid)
+				clog.Infof(ctx,
+					"Removing orchestrator=%v from session list, ethaddress=0x%v, ip-address=%v",
+					sess.Transcoder(),
+					ethcommon.Bytes2Hex(sess.OrchestratorInfo.TicketParams.Recipient),
+					sess.OrchestratorInfo.Transcoder,
+				)
```

Comment:
So this won't be a public log? `clog.Infof`?

Reply:
Oops good catch - this should be public log. Will fix.